subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -440,18 +442,10 @@ static void sw_switch(u8_t dir, u8_t phy_curr, u8_t flags_curr, u8_t phy_next,
 
 #if defined(CONFIG_SOC_NRF52840)
 		if (1) {
-			u8_t ppi_en =
-				HAL_SW_SWITCH_RADIO_ENABLE_S2_PPI(
-					sw_tifs_toggle);
 			u8_t ppi_dis =
 				HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI(
 					sw_tifs_toggle);
 
-			HAL_SW_SWITCH_RADIO_ENABLE_PPI_REGISTER_EVT(
```

Comment:
The reason to remove the invalidation is that we are about to TX (and later switch to RX). By design of the sw_tifs_toggle, the PPI for S2 cannot be in the active group (i.e. it belongs to the "other" group)

Reply:
@ioannisg I remember to have implemented the same (adding and removing the PPI in group), but for reasons unknown I could not get the desired behavior, hence, initialized the group (with 2 sets) once per event, and registered or unregistered in sw_switch.

I see, you have it working with only adding/removing from task group, then I dont see a necessity to explicitly unregistering/reset of the PPI channel.